.github/workflows/codex-review-flag.yml

```
@@ -6,39 +6,56 @@ on:
   pull_request:
     types: [opened, synchronize]
     branches: [main, develop]
+  pull_request_review:
+    types: [submitted]
+    branches: [main, develop]
```

actionlint (1.7.12)

[error] "branches" filter is not available for pull_request_review event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)